src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/ValidateVariants.java

```diff
@@ -209,6 +209,16 @@ public void onTraversalStart() {
             genomeLocSortedSet = new GenomeLocSortedSet(new GenomeLocParser(seqDictionary));
         }
         validationTypes = calculateValidationTypesToApply(excludeTypes);
+
+        //warn user if certain requested validations cannot be done due to lack of arguments
+        if(dbsnp.dbsnp == null && validationTypes.contains(ValidationType.ALL))
```

Comment:
Hmm... it seems like the conditions for these would want to be `validationTypes.contains(All) || validationTypes.contains(IDs)`. It seems like if the user disables REF validation types but forgets to do the same for IDs then it should still warn in this case.